src/iceberg/CMakeLists.txt

```diff
@@ -36,7 +37,6 @@ set(ICEBERG_SOURCES
     transform.cc
     transform_function.cc
     type.cc
-    snapshot.cc
```

Comment:
Not a CPP guy, but why is `snapshot.cc` removed here? 🤔

Reply:
It is duplicate at line 32 :)